types/office-js-preview/index.d.ts

```diff
@@ -13215,6 +13278,40 @@ declare namespace Office {
          * @param itemId - The Exchange Web Services (EWS) identifier for an existing calendar appointment.
          */
         displayAppointmentForm(itemId: string): void;
+        /**
+         * Displays an existing calendar appointment.
+         *
+         * The `displayAppointmentFormAsync` method opens an existing calendar appointment in a new window on the desktop or in a dialog box on
+         * mobile devices.
+         *
+         * In Outlook on Mac, you can use this method to display a single appointment that is not part of a recurring series, or the
+         * master appointment of a recurring series, but you cannot display an instance of the series.
```

Comment:
Consider breaking this into two sentences.
```suggestion
         * master appointment of a recurring series. You cannot display a single instance of a recurring series.
```

Reply:
(timwan) as an aside, this is incredibly interesting to me. I didn't know that.